src/org/usfirst/frc4904/sovereignty/PIDContainer.java

```diff
@@ -0,0 +1,60 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PIDContainer {
+	private final Map<String, Double> values;
+	private final String system;
+	private final boolean locked;
+
+	public static interface PIDContainerModifier {
+		void pushValue(String key, double value);
+
+		double pullValue(String key);
+	}
+
+	// TODO: Consider renaming @ajnadel
+	public static enum PIDValueType {
+		P, I, D, F, SETPOINT, TOLERANCE;
+		public String getKey() {
+			return name();
+		}
+	}
+
+	public PIDContainer(Map<String, Double> values, String system, boolean locked) {
+		this.system = system;
+		this.values = values;
+		this.locked = locked;
+	}
+
+	public PIDContainer(String system, boolean locked) {
+		this(new HashMap<String, Double>(), system, locked);
```

Comment:
new HashMap<>()

Reply:
Respectfully I don't believe it's fair to burden the developer with so much mental work. You don't know how tired the developer might be, or perhaps their particular batch of Coffea arabica's caffeine content was particularly low and thus they aren't as sharp as usual. 

To reduce the burden to the developer, we need to stand up and say NO to syntactic sugar. Though this is only the first step, I think we need to strip back the sickeningly sweet layers to reveal to the developer what is truly going on in the system, until we get back to our core of 5-bit Baudot codes or "real computing" as I like to refer to it. This revolution starts by being explicit in the typing of HashMaps, but it does not end there. No, this will go all the way to the top.